src/page-modules/assistant/layout.tsx

```diff
@@ -246,15 +248,22 @@ function AssistantLayout({ children, tripQuery }: AssistantLayoutProps) {
                 : t(PageText.Assistant.search.buttons.alternatives.more)
             }
             className={style.button}
-            mode="interactive_2"
+            mode={
+              orgId === 'fram' && !isDarkMode
```

Comment:
Should only this one check for dark mode?

Reply:
Yes. The interactive_2 button both looks good and meets the UU requirements in darkmode.  Therefore, the  interactive_2--light-outline should only be used in light mode.